benchmarks/torchbench_model.py

```diff
@@ -255,7 +255,8 @@ def set_up(self):
     if self.benchmark_experiment.xla:
       # First, move the model and the inputs to CPU.
       # This avoids having dupplicated data on CUDA.
-      if self.is_accelerator_cuda():
+      keep_model_data_on_cuda = self.benchmark_experiment.xla_take_cuda_model_and_data
+      if self.is_accelerator_cuda() and not keep_model_data_on_cuda:
```

Comment:
Could you leave a comment here, explaining why we need this flag?

Reply:
Done, added it in the help message